NEWS.rst

```diff
@@ -10,7 +10,7 @@ Bugfixes
 --------
 
 - twisted.internet.defer.inlineCallbacks has an improved type annotation, to avoid typing errors when it is used on a function which returns a non-None result. (#10231)
-- trial.runner.filenameToModule now sets the correct module.__name__ and sys.modules key (#10230)
+- trial.runner.filenameToModule now sets the correct `module.__name__` and `sys.modules` key (#10230)
```

Comment:
```suggestion
- trial.runner.filenameToModule now sets the correct ``module.__name__`` and ``sys.modules`` key (#10230)
```

Reply:
Double for RST?

Reply:
I don't know. But I will use the double back-ticks for the final release. Thanks!